OneMore/Commands/Tagging/HashtagScanner.cs

```diff
@@ -380,7 +380,9 @@ private async Task<bool> ScanPage(
 
 			// saved tags will be in document-order but not have DocumentOrder set,
 			// we can rely on tag + objectID to continue resolving
-			var saved = provider.ReadPageTags(pageID);
+
+			// in case of empty candidate list do not read page tags as it throws exception
+			var saved = (candidates.Count() == 0) ? null : provider.ReadPageTags(pageID);
```

Comment:
This break the case where there are no longer hashtags on the page but there used to be, and now the scanner will not delete the old hashtags in the db.